README.md

```diff
@@ -17,7 +17,7 @@ To build this plugin you need to set up following environments variable:
 - `CLIENT_REDIRECT_URI` - URI to redirect back from AccelByte Login Website to WordPress website
 - `APP_SECURITY_CLIENT_ID` - Client ID from IAM for WordPress application in browser
 - `JUSTICE_PUBLISHER_NAMESPACE` - Default namespace
-- `JUSTICE_APIGATEWAY_BASE_URL` - URL of your AccelByte APIGateway
+- `JUSTICE_BASE_URL` - URL of your AccelByte APIGateway
```

Comment:
We don't use API Gateway anymore, how if we change the description to "Base URL of Accelbyte services"?